README.md

```diff
@@ -51,7 +53,7 @@ If both are specified, the node will pick the action (on or off) from the messag
 ```
 
 ### tradfri-monitor
-This node will send messages when a device or group is updated or removed. The message will contain all available details for the device or group. Due to how this system works these update messages are also sent when reconnecting to the gateway after a flow deploy.
+This node will send messages when a device or group is updated or removed. Examples for updates are buttons presses, change of brightness or a switch being turned on. The message will contain all available details for the device or group. Due to how this system works these update messages are also sent when reconnecting to the gateway after a flow deploy.
```

Comment:
I'm actually still unsure if I can build support for switches being toggled or buttons being pressed. So updates are usually state changes of any connected devices (lights being turned on, brightness being changed, blinds opening or closing, etc.), but not the actual buttons being clicked.
I'll need to finish work on #21 before this would work.

Reply:
Fair point. What I meant is that an event is triggered when a button is pressed, but you are right that the information "which button" is not exposed yet. I'll remove it, so people do not have wrong expectations.